### DISCUSSION TITLE
Improvements to render preview

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6513] [Ideas]
If you are adjusting a parameter of an FX node, OT will only update the render preview after the adjustment has stopped.
Compared to a dedicated compositor such as After Effects (which is the industry standard in the Japanese animation industry), this feels far more sluggish, even if OT's ability to render the preview itself is quick.

I think it would be a better experience if OT were to attempt to continuously update while parameters are being adjusted in order to give much more frequent feedback to the user. 
Dynamically downscaling the preview render during this continuous update could also increase the speed and responsiveness by reducing the time to render the frame. When the user stops adjusting the parameter, then the full resolution frame would be rendered.

One additional request would be an option to render multiple preview frames simultaneously depending on the user's number of threads. Instead of just rendering one preview frame, the user could choose a number of additional frames to render in advance.

**Comment by flurick** [comment]
The second part seems to be implemented now with https://github.com/opentoonz/opentoonz/pull/4712 (?)
